UI/obs-app.cpp

```diff
@@ -2467,6 +2495,10 @@ int main(int argc, char *argv[])
 
 	base_get_log_handler(&def_log_handler, nullptr);
 
+#if defined(__FreeBSD__)
+	move_to_xdg();
```

Comment:
Wouldn't this be optionally conditional on whether portable mode is enabled as a build flag and portable mode _not_ being selected?

Because with portable mode enabled and selected, only config files relative to OBS' location are to be loaded. I'd consider it incorrect behaviour if the app interferes with configuration files that it should not even be aware of/not even consider in that case.

If I understood it right, it's only when _not_ running in portable mode that the new XDG-influenced location would come into play.

Reply:
It turns out that the FreeBSD cmake bits don't have an option to enable portable mode at the moment - it seems there are a number of changes that need to be brought over from the Linux config (e.g. #11410).

I can follow up with a change to enable portable mode builds on FreeBSD and call this only when portable mode is not selected.

Reply:
If there's no way to enable it on FreeBSD than this should indeed be sufficient.